### DISCUSSION TITLE
Python 早就落伍了！AI 權威 LeCun 直言：深度學習需要更靈活的程式語言

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#597] [Ideas]
現今人工智慧；機器學習大多都是以python 為基礎建構的比如anaconda、tensorflow等機器學習架構。
而現在隨著機器學習技術的進步，對於有高運算能力的程式語言與硬體設備也越來越重要
也因此除了python之外，另一種程式語言Julia涵蓋了C 的速度、Matlab 的數學表徵、Python 的通用編程與 Shell 的膠水命令行，同時也支持 Google 的 TPU。
在舊金山舉辦的國際固態電路會議（International Solid State Circuits Conference）指出深度學習的未來必須達成
1.AI 將落實於手機上：使得AI計算程式普及化
2.網路審查工作為因 AI 而更精準：AI將能精準審查網路
3.AI 想走進人類生活，得先解決「晶片」、「電池」的限制：突破硬體的限制
4. AI 未來將具有「常識」：隨著軟硬體的進步，新的AI功能也會被發掘

https://buzzorange.com/techorange/2019/02/19/python-is-old/?fbclid=IwAR3xdD9dzs75EskLL09U6FmUDxUpyv6aEttgv_UXJxMBUtw4moVtSTZDHK0

**Comment by handsomeniko** [comment]
確實可以期待一下

**Comment by Lynn12378** [comment]
難以想像未來Python被取代的一天

**Comment by Jay-722** [comment]
將來有越來越多程式要學的感覺

**Comment by Sazmafuji** [comment]
好多程式要學

**Comment by bloodtrail0511** [comment]
科技發展得比自己學得還快，心累..

**Comment by ThomasMeng0515** [comment]
Python都沒有學到很精熟，感覺又要學新的語言了QQ

**Comment by jcpan821** [comment]
每種程式語言都有自己的特色，如何靈活運用這些語言似乎成為未來的趨勢

**Comment by etin2468** [comment]
我覺得應該還不會這麼快就被取代，雖然說這個Julia看起來涵蓋很多語言的優點，但集成度應該還不如python

**Comment by d26488399** [comment]
可以期待未來有更多好用的程式語言

**Comment by yucheng1028** [comment]
好像要學習更多元的程式了

**Comment by leon10456** [comment]
感謝樓主的分享

**Comment by liangyinglly** [comment]
真的好不容易接觸到python 就有越來越多語言崛起
但我相信精通python後學其他語言也不會太困難

**Comment by HsuChen88** [comment]
每種語言都有它的特點
基本上看目標的特性來決定要用哪種語言寫

**Comment by Jimmy911218** [comment]
謝謝樓主分享

**Comment by evanchou1204** [comment]
看到越來越多程式語言真的是頭很痛

**Comment by Yunhans** [comment]
哇我連python都還沒學好就要落伍了好慘

**Comment by Pietra1226** [comment]
其實短時間內python應該還不會落伍，現階段很多都是用他開發的，但多學點確實是好事

**Comment by Tuoresa** [comment]
這樣的迅速發展真的能帶動科技與各方面產業！但是喔買尬 感覺程式語言真的要多到學不完了，還容易搞混哈哈

**Comment by dabbie109303501** [comment]
在全民都在學python的現在，要取代掉它也是蠻困難的

**Comment by yuxinlin93** [comment]
才學了一個語言，馬上又被取代，似乎要學到死前一秒才夠用

**Comment by wangtyphoon** [comment]
我個人還是覺得學得精比學得多有用，如果單只是要做表層應用python已經很棒啦!

**Comment by Rodrigo-dev-bit** [comment]
我覺得每個程式語言有他自己的獨特的用法，所以python應該不會被完全取代的
